### DISCUSSION TITLE
App translation - Translating of Title

### DISCUSSION THREAD

**Discussion by shift18** [deckerst/aves#1161] [Q&A] [answered]
So, while combing through the Swedish translation I suddenly thought that referring to filenames as titles was weird, i  change the translation from 'titel' (swedish for title) to 'filnamn' (filename).
Now some times has passed and I'm starting to doubt this decision, I've also been trying to ask my self, what would most easily understod by a none techie.
Now, I don't really know what they'd actually think but my guess is that they probably would find 'titel' more easily understandable, so I'm now thinking that It's better to revert all strings mentioning filnamn back title even though I've never seen 'titel'(title) used in place of 'filnamn' (filename) when it comes to digital photos/Images.

So, what do you guys think?
Should i just revert back to 'titel' or should i forge ahead and re-translate all string mentioning 'titel'?

Source strings
Edit title & description
Untitled
Extract from title
Show title filter
Search titles

**Comment by deckerst** [comment, marked as answer]
"Title" is not the same as "filename", and I used "title" on purpose. "Title" is used because it shows the title metadata, and falls back to the file name if the metadata is missing. In other words it's a composite field, not simply the file name.

In general, please stick to the original English when in doubt.

**Reply by shift18** [reply]
I see, title primarily refers to a field in the metadata of each file.
I'll revert all fields back to title.

Thank you for your insight @deckerst
